docs/AboutUs.html

```diff
@@ -31,8 +31,12 @@ <h2 id="project-team"><a class="link" href="#project-team">Project Team</a></h2>
 <div class="sectionbody">
 <div class="sect2">
 <h3 id="adeeb-ashraf"><a class="link" href="#adeeb-ashraf">Adeeb Ashraf</a></h3>
```

Comment:
AboutUs.html is rendered from AboutUs.adoc. Please do not change it.

Reply:
My AboutUs.adoc is exactly the same as yours it just not rendering properly..